sender-stackdriver/src/test/java/zipkin2/reporter/stackdriver/internal/UnaryClientCallTest.java

```diff
@@ -46,32 +47,37 @@
   static class BatchWriteSpansCall extends UnaryClientCall<BatchWriteSpansRequest, Empty> {
     final Channel channel;
 
-    BatchWriteSpansCall(Channel channel, BatchWriteSpansRequest request) {
-      super(channel, TraceServiceGrpc.getBatchWriteSpansMethod(), DEFAULT, request);
+    BatchWriteSpansCall(Channel channel, BatchWriteSpansRequest request, long serverResponseTimeout) {
+      super(channel, TraceServiceGrpc.getBatchWriteSpansMethod(), DEFAULT, request, serverResponseTimeout);
       this.channel = channel;
     }
 
     @Override
     public BatchWriteSpansCall clone() {
-      return new BatchWriteSpansCall(channel, request());
+      return new BatchWriteSpansCall(channel, request(), DEFAULT_SERVER_TIMEOUT_MS);
     }
   }
 
   BatchWriteSpansCall call;
 
   @Before
-  public void setUp() throws Throwable {
+  public void setUp() {
     server.getServiceRegistry().addService(traceService);
-    call = new BatchWriteSpansCall(server.getChannel(), BatchWriteSpansRequest.newBuilder().build());
+    call = new BatchWriteSpansCall(server.getChannel(), BatchWriteSpansRequest.newBuilder().build(), DEFAULT_SERVER_TIMEOUT_MS);
+  }
```

Comment:
there's fuzz in this file. for example, this turns into a redundant thing below. also there are bunch of accidental indents...

```
  @Before
  public void setUp() {
    server.getServiceRegistry().addService(traceService);
    call = new BatchWriteSpansCall(server.getChannel(), BatchWriteSpansRequest.newBuilder().build(), DEFAULT_SERVER_TIMEOUT_MS);
  }

  public void setUp(long serverResponseTimeout) {
    server.getServiceRegistry().addService(traceService);
    call = new BatchWriteSpansCall(server.getChannel(), BatchWriteSpansRequest.newBuilder().build(), serverResponseTimeout);
}
```

Reply:
Are you saying that these two "identical" method calls should be removed? At the moment, the default timeout is 5 seconds, if you remove this configuration, the test execution will take 5 seconds + timeout. Is it works?

Reply:
ah sorry you are right, I didn't notice the subtle difference. I would dispatch the setup method or just inline `setUp(long)` as it causes a mental glitch (in me.. but maybe only me?)